### PR TITLE
Oc 343 review comments

### PR DESCRIPTION
**Pull request by ZaraGlynn**

### CODE REVIEW

File: src/items/model.ts [hunk 1]
```diff
@@ -13,6 +13,19 @@ import { geoJSONToGeom } from '../map/util';
 import { changeS3ProtectionLevel } from '../utils/AWSHelper';
 import { dbgeoparse } from '../utils/dbgeo';
 
+/**
+ *
+ * Get all items by inputQuery, or byField and fieldValue, or by userId
+ *
+ * @param limit: string
+ * @param offset: string
+ * @param isAdmin: boolea
```
Comment (matthewberryman) [inline src/items/model.ts]: missing n

File: src/items/model.ts [hunk 2]
```diff
@@ -170,6 +194,15 @@ export const getItemBy = async (field, value, isAdmin: boolean = false, isContri
...
 };
 
+/**
+ *
+ * Update an item
+ *
+ * @param requestBody: object
+ * @param isAdmin: boolean
+ * @param userId: string
+ * @returns body: success, updated_key, id
```
Comment (Dan-Wood) [inline src/items/model.ts]: @returns body: { success: boolean, updated_key: string , id: number }

File: src/pages/search.ts [hunk 3]
```diff
@@ -3,6 +3,13 @@ import { badRequestResponse, successResponse } from '../common';
 import { db } from '../databaseConnect';
 import Joi from '@hapi/joi';
 
+/**
+ *
+ * Get a result by searching over the items, collections and profiles tables
+ *
+ * @param event.body.string
```
Comment (Dan-Wood) [inline src/pages/search.ts]: body: string?

File: src/shortPaths/shortPaths.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ import Joi from '@hapi/joi';
  * Get an item/profile/collection by it's id or its short path
  *
  * @param event
+ * @return { Promise<APIGatewayProxyResult> } JSON object with body:short_paths
```
Comment (Dan-Wood) [inline src/shortPaths/shortPaths.ts]: body: { short_paths: string[] } ?

File: src/profiles/profiles.ts [hunk 5]
```diff
@@ -133,7 +136,7 @@ export const update = async (event: APIGatewayProxyEvent): Promise<APIGatewayPro
  *
  * @param event {APIGatewayEvent}
  *
- * @returns { Promise<APIGatewayProxyResult> } true
+ * @returns { Promise<APIGatewayProxyResult> } body:true
```
Comment (Dan-Wood) [inline src/profiles/profiles.ts]: body: { boolean }

It's not always true?

File: src/profiles/profiles.ts [hunk 6]
```diff
@@ -62,6 +63,7 @@ export const get = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyR
  * Create a profile
  *
  * @param event
+ * @returns { Promise<APIGatewayProxyResult> } JSON object with body:success, id
```
Comment (Dan-Wood) [inline src/profiles/profiles.ts]: body: { success: boolean, id: number }

File: src/profiles/profiles.ts [hunk 7]
```diff
@@ -14,6 +14,7 @@ import { insertProfile, updateProfile, deleteUserProfile } from './model';
  * Get profile(s) by either it's id, uuid or search by full_name
  *
  * @param event
+ * @returns { Promise<APIGatewayProxyResult> } JSON object with body:profile
```
Comment (Dan-Wood) [inline src/profiles/profiles.ts]: body: { profile }

File: src/items/items.ts [hunk 8]
```diff
@@ -160,7 +160,7 @@ export const getByPerson = async (event: APIGatewayEvent, context: Context): Pro
  * @param event {APIGatewayEvent}
  * @param context {Promise<APIGatewayProxyResult>}
  *
- * @returns { Promise<APIGatewayProxyResult> } JSON object with body:updatedItem - an items s3_key and status
+ * @returns { Promise<APIGatewayProxyResult> } JSON object with body:{ updatedItem }
```
Comment (Dan-Wood) [inline src/items/items.ts]: You could define updated item a lot better.

File: src/items/items.ts [hunk 9]
```diff
@@ -190,7 +190,7 @@ export const changeStatus = async (event: APIGatewayEvent, context: Context): Pr
  *
  * @param event {APIGatewayEvent}
  *
- * @returns { Promise<APIGatewayProxyResult> } tags:tags an array of tags
+ * @returns { Promise<APIGatewayProxyResult> } tags:{ tags: array }
```
Comment (Dan-Wood) [inline src/items/items.ts]: Array of? [{id: number, tag_name: string}] ?

File: src/items/items.ts [hunk 10]
```diff
@@ -105,7 +105,7 @@ export const getByTag = async (event: APIGatewayEvent, context: Context): Promis
  * @param event {APIGatewayEvent}
  * @param context {Promise<APIGatewayProxyResult>}
  *
- * @returns { Promise<APIGatewayProxyResult> } TopoJSON object with data.objects.output.geometries
+ * @returns { Promise<APIGatewayProxyResult> } TopoJSON object with data:{ objects.output.geometries }
```
Comment (Dan-Wood) [inline src/items/items.ts]: Not really defining whats in the geometries object.

File: src/collections/collections.ts [hunk 11]
```diff
@@ -345,7 +345,7 @@ export const getCollectionsByItem = async (event: APIGatewayEvent, context: Cont
  *
  * @param event {APIGatewayEvent}
  *
- * @returns { Promise<APIGatewayProxyResult> } JSON object with body:collections
+ * @returns { Promise<APIGatewayProxyResult> } JSON object with body:{ collections }
```
Comment (Dan-Wood) [inline src/collections/collections.ts]: collection[] ? Array<collection>?

File: src/announcements/announcements.ts [hunk 12]
```diff
@@ -9,7 +9,7 @@ import { limitQuery } from '../utils/queryHelpers';
  * Get an announcement by its id
  *
  * @param event
- * @returns { Promise<APIGatewayProxyResult> } JSON object with body:announcements
+ * @returns { Promise<APIGatewayProxyResult> } JSON object with body:{ announcements }
```
Comment (Dan-Wood) [inline src/announcements/announcements.ts]: announcement[], Array<announcement> ? Or just

{title: string, description: string etc}[]

**Comment by matthewberryman** [comment]
stale

**Pull request closed without merging**